pkg/kubectl/kubectl.go

```diff
@@ -33,6 +33,11 @@ func Execute(c Client, args ...string) (string, error) {
 	return c.Execute(args...)
 }
 
+// ExecuteJSON execute kubectl <args> and returns the combined json stdout and err output.
+func ExecuteJSON(c client, args ...string) (string, error) {
+	return c.Execute(append(args, "-ojson"))
+}
```

Comment:
It'd be nice if we could unmarshal as well in the function, may need to have the struct `interface{}` as first parameter.